apps/root/src/common/components/token-selector/index.tsx

```diff
@@ -112,31 +129,38 @@ const TokenSelector = ({ handleChange, selectedToken }: TokenSelectorProps) => {
     [balances]
   );
 
-  const items: OptionWithKeyAndToken[] = React.useMemo(
-    () =>
-      orderBy(
-        [
-          ...availableTokens,
-          ...(customToken
-            ? [
-                {
-                  ...customToken,
-                  key: customToken.token.address,
-                },
-              ]
-            : []),
-        ],
-        [({ balanceUsd, token }) => parseFloat(formatUnits(balanceUsd || 0n, token.decimals + 18))],
-        ['desc']
-      ),
-    [availableTokens, customToken]
-  );
+  const items: OptionWithKeyAndToken[] = React.useMemo(() => {
+    const parsedWithCustomTokens = availableTokens.map((tokenOption) => ({
+      ...tokenOption,
+      token: {
+        ...tokenOption.token,
+        isCustomToken:
+          !!customTokens[`${tokenOption.token.chainId}-${tokenOption.token.address.toLowerCase()}` as TokenListId],
+      },
+    }));
+
+    return orderBy(
+      parsedWithCustomTokens,
+      [({ balanceUsd, token }) => parseFloat(formatUnits(balanceUsd || 0n, token.decimals + 18))],
```

Comment:
we can use parseUsdPrice here right?

Reply:
With no price available in the component, I've choosen `formatUsdAmount` 😃